app/routes.py

```diff
@@ -1 +1,117 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import datetime
+import os  
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+def validate_tasks(task_id):
+    try:
+        task_id = int(task_id)
+    except:
+        abort(make_response({"details":f"{task_id} invalid data"}, 400))
+
+    task = Task.query.get(task_id)
+
+    if not task:
+        abort(make_response({"message":f"task {task_id} not found"}, 404))
+    
+    return task 
+
+
+@tasks_bp.route("", methods=["POST"])
+def create_tasks():
+    try:
+        request_body = request.get_json()
+        new_task = Task(
+            title=request_body["title"],
+            description=request_body["description"]
+            )
+    except KeyError:
+        return jsonify({"details": "Invalid data"}), 400
```

Comment:
Good validation with a try-except, but you should indicate what data is invalid.